src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -217,14 +211,50 @@ export default class PhotoField extends React.Component {
       ).catch(() => {
         // rather not use the promise here, but seems better than trying to pass
         // a blur function
-        // this.props.onBlur(`${this.props.idSchema.$id}_${idx}`);
+        this.props.onBlur(this.props.idSchema.$id);
       });
-
-      this.setState({ warningMessage: null });
     });
   }
 
+  onChangeNoCropping = (files) => {
+    const file = files[0];
+    const filePath = this.props.idSchema.$id.split('_').slice(1);
+
+    this.screenReaderPath = true;
+
+    const reader = new FileReader();
+    reader.readAsDataURL(file);
+    reader.onload = () => {
+      loadImage(reader.result)
+        .then((img) => {
+          if (!isSquareImage(img)) {
+            this.props.onChange({
+              errorMessage: 'The photo you uploaded is not a square photo. Please upload a new one that fits the requirements.'
```

Comment:
We may not need to require loading a square image since the cropper crops the image into a square. Most images taken by phones, etc are not square and it might be a bad experience to require cropping before loading it.